Tests/SwiftParserTest/Assertions.swift

```diff
@@ -604,7 +604,9 @@ func assertParse<S: SyntaxProtocol>(
   }
 
   // Applying Fix-Its
-  if let expectedFixedSource = expectedFixedSource {
+  if diags.contains(where: { !$0.fixIts.isEmpty }) && expectedFixedSource == nil {
```

Comment:
I think this should be checking whether the expected diagnostics contain any Fix-Its. That way you’ll only get a single assertion failure if a test case starts generating an unexpected Fix-It, instead of 2 for the unexpected Fix-It + the missing `fixedSource`.

Reply:
Hmm I'm not sure I understand. 
If we change to `expectedDiagnostics.contains(where: { !$0.fixIts.isEmpty }) && expectedFixedSource == nil` we still get two assertions if there is multiple Fix-Its in the list of diagnostics. 

We need to compare the diagnostics Fix-Its with the expected diagnostics Fix-Its.

Reply:
I understand now. Thanks for the explanation 😁

Reply:
You’re welcome 😊